### DISCUSSION TITLE
fonts: B612 not shown in dropdown

### DISCUSSION THREAD

**Discussion by stefangweichinger** [Eugeny/tabby#9246] [General]
I have B612 fonts installed. https://b612-font.com/
The fonts are usable in Libre Office Writer, Tilix etc .. GNOME "Fonts" shows them, but Tabby does not offer/show them.
Any reason for that?

**Comment by Eugeny** [comment]
Do you have `B612 Mono` installed as well? Tabby only shows fonts tagged as monospace.

**Reply by stefangweichinger** [reply]
Yes, I forgot to mention, sorry.

I see installed `B612 Mono` in Bold, Bold Italic, Regular Italic ...  and there is a plain `B612 Mono`.

Maybe some simple "grep-error" somewhere? Thanks.

**Reply by stefangweichinger** [reply]
Maybe that font is not tagged as monospace? I will try to find out.

**Reply by stefangweichinger** [reply]
I find [this](https://askubuntu.com/questions/1441908/monospace-font-is-not-recognized-as-a-monospace-font).

`fc-match monospace` does only return ONE font, this can't be right.

No hurries from my side, this is only cosmetic and a "nice-to-have issue", nothing important.

Thanks for your work, @Eugeny !

**Comment by stefangweichinger** [comment]
maybe [this](https://github.com/polarsys/b612/issues/19)

installed the mentioned variant from https://github.com/ekaitz-zarraga/b612

shows up in tabby! solved.
